test/integration/Login.spec.ts

```diff
@@ -0,0 +1,73 @@
+import {login, register} from './pages/Login';
+import {AppModule} from '../../src/app.module';
+import {Test, TestingModule} from '@nestjs/testing';
+
+import {
+  CREATE_ADMIN_DTO,
+  CREATE_USER_DTO_TEST_OBJ,
+  BAD_LOGIN_AUTHENTICATION,
+  ADMIN_LOGIN_AUTHENTICATION,
+  WRONG_EMAIL_AUTHENTICATION
+} from '../constants/users-test.constant';
+import {DatabaseService} from '../../src/database/database.service';
+
+describe('Login and Logout', () => {
+  let databaseService: DatabaseService;
+
+  beforeAll(async () => {
+    const moduleFixture: TestingModule = await Test.createTestingModule({
+      imports: [AppModule],
+      providers: [DatabaseService]
+    }).compile();
+
+    databaseService = moduleFixture.get<DatabaseService>(DatabaseService);
+  });
+  beforeEach(async () => {
+    await page.goto('http:/localhost:8000/');
+  });
+  afterEach(async () => {
+    await page.evaluate(() => {
+      localStorage.clear();
+    });
+  });
+
+  it('Login working', async () => {
```

Comment:
`describe('Login'....` block would be helpful to group these

Reply:
created separate Login and Logout describe block